### PR TITLE
Copy updates for US_PA Admissions -> Recommitments

### LINKED ISSUE #957: Change "Admissions" to "Recommitments" in PA lantern dash
**What needs to be done? Why does it need to be done?**
We are showing board decisions on the PA Lantern dashboard, which represents "recommitments," so saying "Admissions" is causing confusing. 

This task involves changing all instances of "Admissions", "Admission", and "Admitted" on the frontend to "Recommitments", "Recommitment", and "Recommitted."

The PA methodology doc is also updated: https://docs.google.com/document/d/1V-HTYgfklb-VmsYmXmzMCBiNa1CUp4QHbseGl1M2VC8/edit#

**Additional context**
Add any other context about the task here.

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

We are showing board decisions on the PA Lantern dashboard, which represents "recommitments," so saying "Admissions" is causing confusing.

Changed all instances of "Admissions", "Admission", and "Admitted" on the frontend to "Recommitments", "Recommitment", and "Recommitted."

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #957

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/RootStore/TenantStore/us_pa_methodology.js [hunk 1]
```diff
@@ -16,120 +16,120 @@
...
     },
   ],
-  admissionsByDistrict: [
+  recommitmentsBySubOffice: [
     {
       id: 1,
       header: `What the "Admission count" chart shows`,
```
Comment (daschi) [inline src/RootStore/TenantStore/us_pa_methodology.js:56]: Should the headers on lines 56 and 61 be updated to recommitment ?

File: src/RootStore/TenantStore/us_pa_methodology.js [hunk 2]
```diff
@@ -16,120 +16,120 @@
...
+      body: `This chart plots people who were recommitted on a board action to an SCI, CCC, or Contract Facility from parole during the time period selected by the user, according to their most serious violation and the total number of violation reports. The user can select the time period for recommitments included in the chart by using the drop-down menu in the upper left corner of the page. The most serious violation and total number of reported violations are determined by looking back at all violation reports filed within one year prior to the last reported violation before their admission. For example, if the last violation a person committed occurred on February 1, 2017, and they were recommitted on January 15, 2018, the most serious violation and number of violations would be determined by looking over the period of February 1, 2016–February 1, 2017. This methodology negates the lag time between the last violation report and the board action, and therefore gives a fuller picture of a person's behavioral record leading up to the admission decision. Reports filed after the admission are not included, since they were presumptively irrelevant to the admission.`,
```
Comment (daschi) [inline src/RootStore/TenantStore/us_pa_methodology.js:35]: I still see a lot of references to `admission` in this file, do these not need to be updated?

File: src/lantern/FiltersBar/AdmissionTypeFilter.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ const AdmissionTypeFilter = () => {
   );
 
   return (
-    <FilterField label="Admission Type">
+    <FilterField label="Recommitment Type">
```
Comment (daschi) [inline src/lantern/FiltersBar/AdmissionTypeFilter.js:46]: If this component is only used for PA maybe it should be renamed?

**Review by daschi — CHANGES_REQUESTED** [review]
I think the filters description for the exported image should also be updated, right now it still formats the filter as "All admission types".

File: src/RootStore/TenantStore/us_pa_methodology.js (hunk 1 above)
Comment (jovergaag) [reply src/RootStore/TenantStore/us_pa_methodology.js:56]: I missed a bunch. Fixed all of the "Admission" instances that I missed.

File: src/lantern/FiltersBar/AdmissionTypeFilter.js (hunk 3 above)
Comment (jovergaag) [reply src/lantern/FiltersBar/AdmissionTypeFilter.js:46]: I sounds like we want to keep this as "Admission Type" and US_PA "Recommitment Type" is the exception to the rule - even though it is the only one currently (as per the convo in #team-lantern)

**Comment by jovergaag** [comment]
> I think the filters description for the exported image should also be updated, right now it still formats the filter as "All admission types".

I totally missed this comment. Thanks for bringing it to my attention! Fixed.

**Review by daschi — APPROVED** [review]

**Pull request merged**